README.md

```diff
@@ -10,7 +10,13 @@ The Gemini API gives you access to Gemini [models](https://ai.google.dev/models/
 3. [Create](https://aistudio.google.com/app/apikey) an API key.
 4. Use a [quickstart](https://github.com/google-gemini/cookbook/blob/main/quickstarts/Prompting.ipynb) for Python, or call the REST API [using curl](https://github.com/google-gemini/cookbook/blob/main/quickstarts/rest/Prompting_REST.ipynb).
 
+## What's New?
+We're excited to show you our latest additions to the Gemini capabilities. Explore them now and [Google AI Developer Forum](https://discuss.ai.google.dev/).
+* [Code execution](https://github.com/google-gemini/cookbook/blob/main/quickstarts/Code_Execution.ipynb): Solve complex teasks by Generating and runing python code based on plain-text instructions. 
```

Comment:
"Solve complex tasks by generating and running python code based on plain-text instructions."